src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Cohost/IIncompatibleProjectNotifier.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.CodeAnalysis.Razor.Cohost;
+
+internal interface IIncompatibleProjectNotifier
+{
+    void NotifyMiscellaneousFile(TextDocument textDocument);
+    void NotifyNullDocument(Project project, string filePath);
```

Comment:
Consider similar name changes to the suggestions I made on `IIncompatiableProjectService`